test/spec/modules/sonobiBidAdapter_spec.js

```diff
@@ -287,9 +286,14 @@ describe('Sonobi adapter tests', () => {
 
       it('should attempt to call bidder for: ' + adUnitName, () => {
         adapter.callBids(adUnit);
+        expect(stubLoadScript.args[0][0].indexOf('vp=tablet')).to.not.equal(-1);
```

Comment:
This will fail in browserstack (as I've learned after merging the 1.0 version of this) since it tests on various devices.  You need to do a fix similar to something like I did here for the `master` branch: https://github.com/prebid/Prebid.js/commit/37671298f93ec1f073dee3d1afcbe15d417875ad